src/main/java/TaskToEventManual.java

```diff
@@ -13,12 +13,12 @@
      * @return an Event scheduled at a user suggested time
      */
     @Override
-    public Event createEventFromTask(Task task, Calendar calendar) {
+    public Event createEventFromTask(Task task, Calendar calendar, EventScheduler eventScheduler) {
         LocalDateTime userSuggestedTime = getUserSuggestedTime();
-        boolean validTime = calendar.checkAvailability(userSuggestedTime);
+        boolean validTime = eventScheduler.checkAvailability(userSuggestedTime, calendar);
         while (!validTime) {
             userSuggestedTime = getUserSuggestedTime();
-            validTime = calendar.checkAvailability(userSuggestedTime);
+            validTime = eventScheduler.checkAvailability(userSuggestedTime, calendar);
```

Comment:
Yep it does that because it is in a while loop

Reply:
Oh yes I missed it.
In the while loop, it might be better if we use a different method to ask for a new time, because we will want to tell the user the time he chose has conflict. The first time asking for a time is just telling the user to input a time. This can be changed after phase 0